### DISCUSSION TITLE
Dark Souls Remastered

### DISCUSSION THREAD

**Discussion by Heliumrich** [skyline-emu/skyline#453] [Game Requests]
Because I like to suffer

Thanks <3

**Comment by APKROB** [comment]
Up i vote also this one 😍

**Comment by Dilson28** [comment]
Realizar um sonho de poder jogar dark souls no celular em qualquer lugar .

**Comment by BigE-hub** [comment]
Looking for any update on torturing myself on Dark Souls for Mobile

**Comment by hhannine** [comment]
I was hoping to try this as well but I'm only getting a black screen on launch.

**Reply by bogomazdmitry** [reply]
Yes, I try to launch, but black screen stops me

**Comment by rainsterm95** [comment]
Now Its playable , but only up to the menu and character design, with a save game i passed this layer but in the gameplay the render is black and very bugged

**Reply by khanh-it** [reply]
@rainsterm95 Can you share your saved file? I also got stuck on the character design menu!

**Comment by APKROB** [comment]
Yeah sure

**Comment by APKROB** [comment]
https://www.mediafire.com/file/j1n25s38aavheiy/DarkSoulsRe01004AB00A260000.zip/file
